Yona/R.generated.swift

```diff
@@ -461,7 +463,7 @@ struct R {
     }
     
     struct dashboard {
-      static var initialViewController: UITabBarController? { return instance.instantiateInitialViewController() as? UITabBarController }
+      static var initialViewController: BaseTabViewController? { return instance.instantiateInitialViewController() as? BaseTabViewController }
       static var instance: UIStoryboard { return UIStoryboard(name: "Dashboard", bundle: _R.hostingBundle) }
```

Comment:
Every time you access any of these properties you will create new instance of the tab bar or the storyboard; why do we need such thing?